cgmml/common/evaluation/CV/evaluation.py

```diff
@@ -51,14 +52,17 @@ def filter_metadata(indata: list, is_standing: bool) -> list:
             continue
 
         # Check if it is a first frame of the scan
-        if int(data[METADATA_ORDER]) != 1:
+        if one_artifact_per_scan and (int(data[METADATA_ORDER]) != 1):
             continue
 
-        output.append(data)
+        key = data[METADATA_SCAN_ID]
+        if not output.get(key):
+            output[key] = []
```

Comment:
To avoid the 2 lines of code, please use a Python [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict)

```suggestion
        if not output.get(key):
            output[key] = []
```